packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
```

Comment:
Do we need to have an outline version of "search" icon? 
CC: @codepretty

Reply:
This actually is the outline version. I'll request a filled one, but it shouldn't hold up this PR